src/main/java/com/vdovich/tbfm/GreetingController.java

```diff
@@ -1,17 +1,19 @@
 package com.vdovich.tbfm;
 
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
```

Comment:
please remove unused import

Reply:
using this import  for callNasaApi method